modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -266,6 +266,10 @@
       {{/isOAuth}}
       {{/authMethods}}
 
+      if (!is.null(accept) && names(headerParams) == "accept")
```

Comment:
This will remove any other header values (such as the api-key).  It should instead update or replace only the Accept header. headerParams['Accept'] <- accept

Reply:
I changed the line to `headerParams['accept'] <- c(accept)` 
one thing I noted is that `headerParams['accept'] ` is not set anywhere